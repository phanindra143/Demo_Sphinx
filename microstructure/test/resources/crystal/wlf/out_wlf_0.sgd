*if,ifprop,eq,2,then
  !c_ij(1,1)=     0.0000
  !c_ij(1,2)=     0.0000
  !c_ij(2,2)=     0.0000
  !c_ij(1,3)=     0.0000
  !c_ij(2,3)=     0.0000
  !c_ij(3,3)=     0.0000
  !c_ij(1,4)=     0.0000
  !c_ij(2,4)=     0.0000
  !c_ij(3,4)=     0.0000
  !c_ij(4,4)=     0.0000
  !c_ij(1,5)=     0.0000
  !c_ij(2,5)=     0.0000
  !c_ij(3,5)=     0.0000
  !c_ij(4,5)=     0.0000
  !c_ij(5,5)=     0.0000
  !c_ij(1,6)=     0.0000
  !c_ij(2,6)=     0.0000
  !c_ij(3,6)=     0.0000
  !c_ij(4,6)=     0.0000
  !c_ij(5,6)=     0.0000
  !c_ij(6,6)=     0.0000
*endif
*if,ifprop,eq,1,then
  !k_ij(1,1)=   100.1255
  !k_ij(1,2)=     0.1565
  !k_ij(2,2)=    99.5863
  !k_ij(1,3)=     0.1178
  !k_ij(2,3)=     0.1971
  !k_ij(3,3)=    99.5733
MP,Kxx,1,  100.12553
MP,Kyy,1,   99.58633
MP,Kzz,1,   99.57328
MP,Kxx,2,   99.58633
MP,Kyy,2,  100.12553
MP,Kzz,2,   99.57328
MP,Kxx,3,   99.58633
MP,Kyy,3,  100.12553
MP,Kzz,3,   99.57328
MP,Kxx,4,  100.12553
MP,Kyy,4,   99.58633
MP,Kzz,4,   99.57328
MP,Kxx,5,  100.12553
MP,Kyy,5,   99.58633
MP,Kzz,5,   99.57328
MP,Kxx,6,   99.58633
MP,Kyy,6,  100.12553
MP,Kzz,6,   99.57328
MP,Kxx,7,   99.58633
MP,Kyy,7,  100.12553
MP,Kzz,7,   99.57328
MP,Kxx,8,  100.12553
MP,Kyy,8,   99.58633
MP,Kzz,8,   99.57328
MP,Kxx,9,   99.57328
MP,Kyy,9,  100.12553
MP,Kzz,9,   99.58633
MP,Kxx,10,  100.12553
MP,Kyy,10,   99.57328
MP,Kzz,10,   99.58633
MP,Kxx,11,  100.12553
MP,Kyy,11,   99.57328
MP,Kzz,11,   99.58633
MP,Kxx,12,   99.57328
MP,Kyy,12,  100.12553
MP,Kzz,12,   99.58633
MP,Kxx,13,   99.57328
MP,Kyy,13,  100.12553
MP,Kzz,13,   99.58633
MP,Kxx,14,  100.12553
MP,Kyy,14,   99.57328
MP,Kzz,14,   99.58633
MP,Kxx,15,  100.12553
MP,Kyy,15,   99.57328
MP,Kzz,15,   99.58633
MP,Kxx,16,   99.57328
MP,Kyy,16,  100.12553
MP,Kzz,16,   99.58633
MP,Kxx,17,   99.58633
MP,Kyy,17,   99.57328
MP,Kzz,17,  100.12553
MP,Kxx,18,   99.57328
MP,Kyy,18,   99.58633
MP,Kzz,18,  100.12553
MP,Kxx,19,   99.57328
MP,Kyy,19,   99.58633
MP,Kzz,19,  100.12553
MP,Kxx,20,   99.58633
MP,Kyy,20,   99.57328
MP,Kzz,20,  100.12553
MP,Kxx,21,   99.58633
MP,Kyy,21,   99.57328
MP,Kzz,21,  100.12553
MP,Kxx,22,   99.57328
MP,Kyy,22,   99.58633
MP,Kzz,22,  100.12553
MP,Kxx,23,   99.57328
MP,Kyy,23,   99.58633
MP,Kzz,23,  100.12553
MP,Kxx,24,   99.58633
MP,Kyy,24,   99.57328
MP,Kzz,24,  100.12553
*endif
