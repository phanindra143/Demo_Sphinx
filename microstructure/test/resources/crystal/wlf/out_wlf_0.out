            ifx                 q_x                 q_y                 q_z     
  /% source: R:/166_320/Ablage_Simulationen/Modellbildung_Simulation_CMC-SiC_[Vorlauf+WiMi+MaVo]/04_04_2017_Modellierung_Matrix_mikro_ditt_[WiMi]/2_Waermeleitfähigket_SiC-Si\Rein_ohne_Poren\30-70
             3.00000      -1564450.69610         -1157.34063          5296.64816
  /%
            ifx                 dtx                 dty                 dtz     
  /%
             3.00000         15625.00000           -14.30827           -71.86113
  /%
            ifx                 q_x                 q_y                 q_z     
  /%
             2.00000         -2491.73870      -1556016.74566          6548.97702
  /%
            ifx                 dtx                 dty                 dtz     
  /%
             2.00000             2.10225         15625.00000           -98.10304
  /%
            ifx                 q_x                 q_y                 q_z     
  /%
             1.00000          4987.61063          5805.52203      -1555808.18929
  /%
            ifx                 dtx                 dty                 dtz     
  /%
             1.00000           -67.85278           -87.72788         15625.00000
  /%
